portal/templates/redesign/base_new.html

```diff
@@ -0,0 +1,215 @@
+{% load staticfiles %}
+{% load app_tags %}
+{% load cache %}
+
+{# CMS #}
+{% load cms_tags sekizai_tags %}
+
+<!DOCTYPE html>
+<html>
+<head>
+    {# CMS #}
+    {% render_block "css" %}
+
+    <title>{% block title %}Code for Life{% endblock %}</title>
+    <meta name="msapplication-config" content="none"/>
+
+    {% block head %}
+    {% endblock head %}
+
+    {% block css %}
+    {% cache 3600 fragment-base_new-css %}
+    <link href='{% static "portal.css" %}' type="text/css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Baloo" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+    {% endcache %}
+    {% endblock css %}
+
+    {% cache 3600 fragment-base_new-javascript %}
+    <script type='text/javascript' src='{% static "portal/js/lib/jquery.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/lib/jquery-ui.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/lib/modernizr.custom.79639.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/common.js" %}'></script>
+    <script type="text/javascript" src='{% static "portal/js/deleteWorkspaces.js" %}'></script>
+    {% endcache %}
+    <script type="text/javascript" src='{% static "portal/js/jquery.placeholder.js" %}'></script>
```

Comment:
Any reason this one is outside the block?  This is how Django caching works:
https://docs.djangoproject.com/en/1.10/topics/cache/